commands/InjectedCommand.java

```diff
@@ -1,58 +1,45 @@
 package org.usfirst.frc4904.standard.commands;
 
+import edu.wpi.first.wpilibj2.command.CommandBase;
 
-import edu.wpi.first.wpilibj.command.Command;
+public abstract class InjectedCommand extends CommandBase {
+	private final CommandBase previous;
 
-public abstract class InjectedCommand extends Command {
-	private final Command previous;
-	
-	public InjectedCommand(Command previous) {
+	public InjectedCommand(String name, CommandBase previous) {
 		super();
```

Comment:
pretty sure that this is an unnecessary `super()` call as it is implicit by the java compiler

check this though with someone else

Reply:
I think `super()` is needed for proper `CommandBase` functionality